README.md

```diff
@@ -171,6 +172,17 @@ def sb = new org.mozilla.fxtest.ServiceBook()
 sb.testProject('kinto')
 ```
 
+## How to run tests
```

Comment:
Would you mind making this a top level heading, otherwise it appears to belong to the ServiceBook section.